chart/values.yaml

```diff
@@ -5408,3 +5452,732 @@ gateway:
+  ## ref: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
+  ## e.g:
+  ## initContainers:
+  ##  - name: your-image-name
+  ##    image: your-image
+  ##    imagePullPolicy: Always
+  ##    command: ['sh', '-c', 'echo "hello world"']
+  ##
+  initContainers: {}
+  ## @section ai-api Service Parameters
+  ##
+  ##
+  service:
+    ## @param aiApi.service.type ai-api service type
+    ##
+    type: ClusterIP
+    ## @param aiApi.service.ports.http ai-api service HTTP port
+    ##
+    ports:
+      http: 80
+    ## Node.js ports to expose
+    ## @param aiApi.service.nodePorts.http Node.js port for HTTP
+    ## NOTE: choose port between <30000-32767>
+    ##
+    nodePorts:
+      http: ""
+    ## @param aiApi.service.clusterIP ai-api service Cluster IP
+    ## e.g.:
+    ## clusterIP: None
+    ##
+    clusterIP: ""
+    ## @param aiApi.service.loadBalancerIP ai-api service Load Balancer IP
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-loadbalancer
+    ##
+    loadBalancerIP: ""
+    ## @param aiApi.service.labelSelectorsOverride Selector for ai-api service
+    ##
+    labelSelectorsOverride: {}
+    ## @param aiApi.service.loadBalancerSourceRanges ai-api service Load Balancer sources
+    ## ref: https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+    ## e.g:
+    ## loadBalancerSourceRanges:
+    ##   - 10.10.10.0/24
+    ##
+    loadBalancerSourceRanges: []
+    ## @param aiApi.service.externalTrafficPolicy ai-api service external traffic policy
+    ## ref http://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip
+    ##
+    externalTrafficPolicy: Cluster
+    ## @param aiApi.service.annotations Additional custom annotations for ai-api service
+    ##
+    annotations: {}
+    ## @param aiApi.service.extraPorts Extra ports to expose in ai-api service (normally used with the `sidecars` value)
+    ##
+    extraPorts: []
+    ## @param aiApi.requestTimeoutMs ai-api request timeout in milliseconds
+    ##
+  requestTimeoutMs: 60000
+  ## @param aiApi.maxResponseLength ai-api max response length in bytes
+  ##
+  maxResponseLength: 10485760
+  ## @param aiApi.bodyJsonLimit ai-api body json limit
+  ##
+  bodyJsonLimit: "5mb"
+  ## @param aiApi.healthCheckServiceTimeout ai-api health check service timeout in milliseconds
+  ##
+  healthCheckServiceTimeout: 60000
+  ## @param aiApi.honeycombDataset ai-api honeycomb dataset
+  ##
+  honeycombDataset: ""
+  ## @param aiApi.honeycombApiKey ai-api honeycomb api key
+  ##
+  honeycombApiKey: "dummy-key"
+  ## @param aiApi.openTelemetryEnabled Enable OpenTelemetry tracing for ai-api
+  ##
+  openTelemetryEnabled: true
+  ## @param aiApi.openTelemetryTracesEndpoint OpenTelemetry traces endpoint for ai-api
+  ##
+  openTelemetryTracesEndpoint: "https://api.honeycomb.io/v1/traces"
```

Comment:
We should delete all these to follow the same approach that we have with other services :)